SimGeneral/TrackingAnalysis/python/simHitTPAssociation_cfi.py

```diff
@@ -19,6 +19,9 @@
   trackingParticleSrc = cms.InputTag('mix', 'MergedTrackTruth')
 )
 
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+trackingPhase2PU140.toModify(simHitTPAssocProducer, simHitSrc = ["g4SimHits:TrackerHitsPixelBarrelLowTof", "g4SimHits:TrackerHitsPixelEndcapLowTof"])
```

Comment:
I think this should be `phase2_tracker` as the collections depend on the detector configuration.